src/redux/stores/orders/ordersActions.ts

```diff
@@ -85,12 +81,7 @@ ApproveParams,
         // const state: RootState = getState() as RootState;
```

Comment:
are those comments still relevant ?

Reply:
Nope, I have removed these comments already in the #88 branch. Will come after this one!